src/tasks/assign-group/assign-group-v2/task.json

```diff
@@ -0,0 +1,122 @@
+{
+  "id": "07D9833F-C02D-4CF6-B10A-EA56CC9C32F0",
+  "name": "PowerPlatformAssignGroup",
+  "friendlyName": "Power Platform Assign Group",
+  "description": "Assign group to target environment with specified security role",
+  "author": "Microsoft",
+  "helpMarkDown": "[More Info](https://aka.ms/buildtoolsdoc)",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "PowerPlatformTool"
+  ],
+  "version": {
+    "Major": "0",
+    "Minor": "1",
+    "Patch": "0"
+  },
+  "preview": false,
+  "minimumAgentVersion": "1.95.0",
+  "instanceNameFormat": "Power Platform Assign Group $(message)",
+  "inputs": [
+    {
+      "name": "authenticationType",
+      "type": "radio",
+      "label": "Authentication type",
+      "defaultValue": "PowerPlatformEnvironment",
+      "required": true,
+      "helpMarkDown": "Authentication via username/password is simpler to set up but won't work if your AAD tenant requires MFA; use Service Principal Name (SPN) if MFA is required.",
+      "options": {
+        "PowerPlatformEnvironment": "Username/password (no MFA support)",
+        "PowerPlatformSPN": "Service Principal/client secret (supports MFA)"
+      }
+    },
+    {
+      "name": "PowerPlatformEnvironment",
+      "type": "connectedService:Generic",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformEnvironment",
+      "helpMarkDown": "Authenticate with your Power Platform environment with username/password. Does not support MFA."
+    },
+    {
+      "name": "PowerPlatformSPN",
+      "type": "connectedService:powerplatform-spn",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformSPN",
+      "helpMarkDown": "Authenticate with your Power Platform environment with an Azure AppID, tenantID and client secret."
+    },
+    {
+      "name": "Environment",
+      "label": "Environment",
+      "type": "string",
+      "defaultValue": "$(BuildTools.EnvironmentUrl)",
+      "required": true,
+      "helpMarkDown": "Environment URL or ID this task targets; default is the pipeline variable that e.g. the CreateEnvironment task has set before."
+    },
+    {
+      "name": "Group",
+      "label": "Group",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "AAD object id of group to assign to target Dataverse environment."
+    },
+    {
+      "name": "GroupName",
+      "label": "Group Name",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "Name of group/team that will created in Dataverse."
+    },
+    {
+      "name": "Role",
+      "label": "Role",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "Security role name or id to assign to the user"
```

Comment:
`"Security role name or id to assign to the user"`
Assign to the _group_?

Reply:
Darn, I thought I had done a complete update on instances of user. Good catch.